base_layer/mmr/src/mutable_mmr.rs

```diff
@@ -209,13 +210,13 @@ where
     fn get_sub_bitmap(&self, leaf_index: LeafIndex, count: usize) -> Result<Bitmap, MerkleMountainRangeError> {
         let mut deleted = self.deleted.clone();
         if leaf_index.0 > 0 {
-            deleted.remove_range_closed(0..u32::try_from(leaf_index.0 - 1).unwrap())
+            deleted.remove_range(0..u32::try_from(leaf_index.0 - 1).unwrap())
```

Comment:
Can we check this unwrap and the arithmetic here?

Reply:
good check, I just rename the api fix